client/models/service-instance.js

```diff
@@ -2,15 +2,6 @@ var request = require('request');
 var util = require('util');
 
 module.exports = function(ServiceInstance) {
-  function applyPatch(target, fileData, callback) {
-    return this._appCommand({
-      cmd: 'patch',
-      target: target,
-      patch: fileData
-    }, callback);
-  }
-  ServiceInstance.prototype.applyPatch = applyPatch;
```

Comment:
The JS api does not change. The same method is not implemented as a remote method on ServiceInstance. https://github.com/strongloop/strong-mesh-models/blob/feature/heroku/common/models/service-instance.js#L428. So instance.applyPatch(...) will continue to work.